packages/webapp/src/routes/__tests__/developer/confirm-off-site-gain.spec.js

```diff
@@ -0,0 +1,223 @@
+import constants from '../../../utils/constants.js'
+
+const url = '/' + constants.routes.DEVELOPER_CONFIRM_OFF_SITE_GAIN
+
+const mockMetricData = {
+  offSiteHabitatBaseline: [
+    {
+      'Broad habitat': 'Rocky shore ',
+      'Habitat type': 'Moderate energy littoral rock - on peat, clay or chalk',
+      'Area (hectares)': 1,
+      'Total habitat units': 'Check Data ⚠',
+      Condition: 'Fairly Good'
+    },
+    { 'Area (hectares)': 1, 'Total habitat units': 1 }
+  ],
+  offSiteHedgeBaseline: [
+    {
+      'Hedgerow type': 'Native hedgerow - associated with bank or ditch',
+      'Length (km)': 3,
+      'Total hedgerow units': 27,
+      Condition: 'Good'
+    },
+    { 'Length (km)': 3, 'Total hedgerow units': 27 }
+  ]
+}
+
+const getNumOfUnits = (data, field1, field2) => (data || []).reduce((prev, item) => {
+  const area = item[field1] && !isNaN(item[field2]) ? item[field2] : 0
+  return prev + area
+}, 0)
+
+describe(url, () => {
+  describe('GET', () => {
+    let viewResult, contextResult
+    const redisMap = new Map()
+    it(`should render the ${url.substring(1)} view`, async () => {
+      const confirmOffsiteGainOptions = require('../../developer/confirm-off-site-gain.js')
+      redisMap.set(constants.redisKeys.DEVELOPER_METRIC_DATA, mockMetricData)
+      const request = {
+        yar: redisMap
+      }
+      const h = {
+        view: (view, context) => {
+          viewResult = view
+          contextResult = context
+        }
+      }
+      await confirmOffsiteGainOptions.default[0].handler(request, h)
+      expect(viewResult).toEqual(constants.views.DEVELOPER_CONFIRM_OFF_SITE_GAIN)
+      expect(contextResult).toBeDefined()
+      expect(getNumOfUnits(mockMetricData.offSiteHabitatBaseline, 'Broad habitat', 'Area (hectares)')).toBeDefined()
+      expect(getNumOfUnits(mockMetricData.offSiteHedgeBaseline, 'Hedgerow type', 'Length (km)')).toBeDefined()
```

Comment:
I suggest improving the test by checking the values are as expected rather than simply being defined.